certbot-postfix/setup.py

```diff
@@ -0,0 +1,59 @@
+import sys
+
+from setuptools import setup
+from setuptools import find_packages
+
+
+version = '0.22.0.dev0'
+
+install_requires = [
+    'acme=={0}'.format(version),
+    'certbot=={0}'.format(version),
+    # For pkg_resources. >=1.0 so pip resolves it to a version cryptography
+    # will tolerate; see #2599:
+    'setuptools>=1.0',
+    'six',
+    'zope.interface',
+]
+
```

Comment:
I looked into doing this via `dependency_links` for some time, it seems that it can't accept a Github link unless `setup.py` is in the root folder. Once it's on `pypi` proper, there should be less of an issue.

I'll try out option 1 for now.

Reply:
Option 1 with a requirements file until `starttls-policy` is shipped sounds the best to me. If you're removing the STARTTLS Everwhere enhancement though, I think you can kick this to that PR.

I spent a little time looking at `dependency_links` and apparently using it requires special flags to `pip` and installing packages in this way is deprecated. See https://github.com/pypa/pip/issues/3939.